### DISCUSSION TITLE
Power Automate Connector

### DISCUSSION THREAD

**Discussion by AnthonyHunt-UoW** [leihs/leihs#2021] [Show and tell]
I have been developing a method of synchronising/maintaining the LEIHS Users using Power Automate (for those of use that don't have admin access to AD or development servers).

As part of this I have developed a number of Power Automate flows and feature. I'm now updating these to make them sharable. The first of which is a Custom Connector for the User/Groups API. A bit rough and ready - but available here.

https://github.com/AnthonyHunt-UoW/LEIHS-Power-Automate-Connector

**Comment by nimaai** [comment]
thanks for sharing @AnthonyHunt-UoW
